source/projects/configure-settings.txt

```diff
@@ -42,6 +42,20 @@ Key Handling Options
 Specifies how the ``_id`` field is populated when new documents are
 inserted into a collection. You can choose one of the following options:
 
```

Comment:
question: in the JIRA it says the order should be GENERATED, SINGLE, then WRAPPED. We moved GENERATED, but any reason why we didn't move SINGLE?